js/click.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+var CarDealer = (function(oldCarDealer){
+  oldCarDealer.activateEvents = function(event){
+
+     var input = document.getElementById('input');
+     var inventory = document.getElementsByClassName('card');
+     var color = 0;
+     var section = document.getElementById('bodWrapper')
+
+     document.addEventListener("click", newClick);
+     input.addEventListener("keyup" , newData);
+              
+              function newClick(event){
+                for (var i=0; i<inventory.length; i++){
+                    var newSel = inventory[i];
+                    color = "#59BFBF";
+                    console.log(newSel)
+                    CarDealer.removeClass(newSel , color);
+
+                    if (event.target === inventory[i] || event.target.closest('div') === inventory[i]){
```

Comment:
You can keep using that `newSel` var you created here, instead of `inventory[i]. Saves a bit of time and makes the code more readable